<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pathLabel_">
       <property name="text">
        <string>Sync folder:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="pathRow_">
       <item>
        <widget class="QLineEdit" name="pathLineEdit_"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseBtn_">
         <property name="text">
          <string>Browseâ€¦</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cloudCheckLabel_">
       <property name="text">
        <string>Cloud check interval (sec):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="cloudCheckSpinBox_">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="refreshLabel_">
       <property name="text">
        <string>Refresh interval (sec):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="refreshSpinBox_">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsRow_">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn_">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn_">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets/>
 <resources/>
 <connections/>
</ui>
